include/librealsense2/dds/dds-device-server.cpp

```diff
@@ -320,12 +328,12 @@ void dds_device_server::init( const std::vector<std::string> &supported_streams_
     }
 }
 
-void dds_device_server::publish_notifications( const topics::raw::device::notifications& notifications_msg )
+void dds_device_server::publish_notification( const topics::raw::device::notification& notification_msg )
 {
-    _dds_notifications_server->send_notifications( notifications_msg );
+    _dds_notifications_server->send_notification( notification_msg );
 }
-void dds_device_server::add_init_msgs( const topics::raw::device::notifications& notifications_msg )
+void dds_device_server::add_init_msgs( const topics::raw::device::notification& notification_msg )
```

Comment:
Shouldn't this function also be singular? `add_init_msg`? BTW, here it's called `msg` but elsewhere `notification`...

Reply:
👍